pkg/option/config.go

```diff
@@ -1128,9 +1128,12 @@ const (
 	// filters to be inserted prior to the cilium filter.
 	TCFilterPriority = "bpf-filter-priority"
 
-	// Flag to enable BGP control plane features
+	// Flag to enable BGPv1 control plane features
 	EnableBGPControlPlane = "enable-bgp-control-plane"
 
+	// Flag to enable BGPv2 control plane features
+	EnableBGPV2ControlPlane = "enable-bgp-v2-control-plane"
```

Comment:
Why not use the same config value, `bgp-v2-api-enabled`, so that nothing has to change for Helm?